test/hotspot/jtreg/serviceability/jvmti/vthread/MethodExitTest/libMethodExitTest.cpp

```diff
@@ -122,7 +122,9 @@ clear_breakpoint(JNIEnv *jni, const char *methodName,
   set_or_clear_breakpoint(jni, JNI_FALSE, methodName, klass, methods, method_count);
 }
 
-static long tls_data = 0;
+static void* tls_data = 0;
+static const void* const tls_data1 = (const void*)0x111;
+static const void* const tls_data2 = (const void*)0x222;
```

Comment:
Was this necessitated by the change to `void*`? If so I had not expected that.

Reply:
I guess I could have changed the logging code to use `"%d", (int)(intptr_t)tls_data` and call it a day, but it would be ugly.